src/main/java/frc/robot/subsystems/swerve/modules/ModuleConstants.java

```diff
@@ -2,18 +2,35 @@
 
 import edu.wpi.first.math.geometry.Rotation2d;
 import frc.robot.constants.LogPaths;
-import frc.robot.subsystems.swerve.SwerveConstants;
 import frc.utils.Conversions;
 import frc.utils.battery.BatteryUtils;
 
-public record ModuleConstants(double wheelDiameterMeters, double couplingRatio, Rotation2d velocityAt12VoltsPerSecond) {
+public record ModuleConstants(
+        String modulesLogPath,
+        double wheelDiameterMeters,
+        double couplingRatio,
+        Rotation2d velocityAt12VoltsPerSecond
+) {
 
-    public ModuleConstants(double wheelDiameterMeters, double couplingRatio, double velocityAt12VoltsMetersPerSecond){
-        this(wheelDiameterMeters, couplingRatio, Conversions.distanceToAngle(velocityAt12VoltsMetersPerSecond, wheelDiameterMeters));
+    public ModuleConstants(
+            String swerveLogPath,
+            String modulesName,
```

Comment:
I dont like this force swerve log path, butt i get why its not connected with module name. maybe consider to get the whole log path outside, or combine them under different name like logPathPrefix